app/tests/unit/validators/common.spec.js

```diff
@@ -2,7 +2,7 @@ const crypto = require('crypto');
 const jestJoi = require('jest-joi');
 expect.extend(jestJoi.matchers);
 
-const { Permissions } = require('../../../src/components/constants');
+const { EMAILREGEX, Permissions } = require('../../../src/components/constants');
```

Comment:
This is part of Kyle's PR. The regex is being backported from BCBox to here so that it remains in alignment - the Joi email definition isn't sufficiently meeting our needs.